test/functional/checkObjectExists.js

```diff
@@ -1,8 +1,6 @@
 const lib = 'NODETKTEST';
 const createLib = `CRTLIB LIB(${lib}) TYPE(*TEST) TEXT('Used to test Node.js toolkit')`;
 const findLib = `SELECT SCHEMA_NAME FROM qsys2.sysschemas WHERE SCHEMA_NAME = '${lib}'`;
-const dataQueue = 'TESTQ';
```

Comment:
../nodejs-itoolkit/test/functional/checkObjectExists.js
    4:7   error  'dataQueue' is assigned a value but never used                                   no-unused-vars